source/_patterns/02-organisms/55-auto-deposit/05-thanks.json

```diff
@@ -2,5 +2,6 @@
 	"deposit_amount": "25.00",
 	"has_donation": true,
 	"is_donation_only": false,
-	"donation_amount": "3.75"
+	"donation_amount": "3.75",
+	"credit_settings_url": "https://www.kiva.org/settings/credit"
```

Comment:
@mcstover - I wonder if we ought to call this "auto_lend_url" even though we are setting the value for now to be the credit setting page, in anticipation that eventually we may re-point it to a better auto-lend page.